package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "research-software-directory",
+  "description": "",
+  "main": "index.js",
```

Mark it as private so npm will refuse publish
```suggestion
  "main": "index.js",
  "private": true,
```